_styles/header.scss

```diff
@@ -175,7 +175,7 @@ header[data-big] {
 
 @media (max-width: $limit-title) {
   .title > *:first-child {
-    font-size: var(--medium);
+    font-size: var(--xxl);
```

Comment:
This is a good temporary fix for the font issue. In my browser, our logo and font still resize in a funny way. I think we should make the logo image also include the `mphoward lab` text so that way they shrink the same. We can handle that in a separate PR though.